docroot/sites/all/modules/features/ilr_forms_public/ilr_forms_public.install

```diff
@@ -0,0 +1,10 @@
+<?php
+
+/**
+ * Deletes entityform field instances needing to be changed
```

Comment:
The comment here is actually important, as this is what drush uses when you verify whether to run a particular update hook. So this should be specific, i.e. "Removes field_fax from ICS mailing list form"